doc/FertiScan/fertiScan-architecture.md

```diff
@@ -168,31 +160,44 @@ erDiagram
     string name_fr
     string name_en
     string symbol
+  }
+    metric{
+    uuid id PK
+    float value
+    boolean edited
+    ENUM metric_type 
+    uuid unit_id FK
+    uuid label_id FK
+  }
+  unit{
+    uuid id PK
+    string unit
+    float to_si_unit
   }
   inspection ||--|| sample :has
   picture_set ||--|{picture : contains
-  inspection ||--o| organization: manufacturer
-  inspection ||--o| organization: company
   fertilizer ||--|| organization: responsable
-  location }|--|| organization: host
-  organization ||--|| location: HQ
+  organization_information ||--|| location: Hosts
   location ||--|| region: defines
   region ||--|| province: apart
   inspection ||--|| fertilizer : about
   inspection }|--|| users :inspect
   inspection ||--o| picture_set :has
   inspection ||--|| label_information : defines
-  label_information ||--|o metric: weight
-  label_information ||--|o metric: density
-  label_information ||--|o metric: volume
   label_information ||--|{ ingredient: has
   label_information ||--|{ guaranteed: has
   label_information ||--|{ micronutrient: has
   label_information ||--|{ specification: has
   label_information ||--|{ sub_label: has
+  label_information ||--o| organization_information: company
+  label_information ||--o| organization_information: manufacturer
+  organization_information ||--|| organization: defines
+  label_information ||--|{ metric: has
   sub_label }o--|| sub_type: defines
   users ||--o{ picture_set: owns
   metric ||--|| unit: defines
+  metric ||--|| metric_type: defines
+
```

Comment:
Wondering if:
micronutrient }o--|| element_compound: is
  guaranteed }o--|| element_compound: is

and also 

metric }o--|| unit: defines

But def `metric ||--|| metric_type: defines` no longer exists.